src/style.scss

```diff
@@ -11,73 +11,79 @@
 @import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
 @import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
 
-// Support for auroElement styles
-// @import './node_modules/@aurodesignsystem/webcorestylesheets/dist/auroElement/auroElement';
-
-.toastButton {
-  width: var(--auro-size-400);
-  height: var(--auro-size-400);
-  padding-right: 0;
-  padding-left: 0;
-  border: 2px solid transparent;
-  margin-left: auto;
-  background-color: transparent;
-
-  &:hover {
-    border: 1px solid var(--auro-color-ui-bkg-hover-on-light);
-    border-radius: 50%;
-    background-color: var(--auro-color-ui-bkg-hover-on-light);
-    cursor: pointer;
-
-  }
-
-}
-
 // container styles - defines the look of the outer custom component
 :host([visible]) {
-  position: fixed;
-  z-index: var(--auro-depth-modal);
-  right: 2%;
-  bottom: 2%;
-  left: 2%;
   display: inline-flex;
-  min-width: 95%;
+  min-width: 100%;
   min-height: 60px;
-  align-items: center;
   padding: var(--auro-size-200) var(--auro-size-150);
   border-radius: 8px;
   background-color: var(--auro-color-brand-gray-400);
   box-shadow: var(--auro-elevation-200);
-  gap: var(--auro-size-200);
   color: var(--auro-color-text-primary-on-dark);
+  gap: var(--auro-size-200);
+  pointer-events: auto;
 
-  @media ($auro-grid-breakpoint-sm <= width < $auro-grid-breakpoint-lg) {
-    right: 1%;
-    left: unset;
+  @media ($auro-grid-breakpoint-sm <=width < $auro-grid-breakpoint-lg) {
     min-width: 33%;
     max-width: 50%;
-
   }
   @media (width >=$auro-grid-breakpoint-lg) {
-    right: 1%;
-    left: unset;
     min-width: 200px;
     max-width: 500px;
   }
 
+  #toastContainer {
```

Comment:
We typically do not use ID as CSS selectors. https://auro.alaskaair.com/css/conventions